Improve automatic install software guide (#23431)
fleetdm/fleet: articles/automatic-software-install-in-fleet.md

Fleet allows its users to upload trusted software installation files to be installed and used on hosts. This installation could be conditioned on a failure of a specific Fleet Policy.

> > Currently, custom packages and Fleet-maintained apps can be automatically installed. macOS App Store apps will be [supported soon](https://github.com/fleetdm/fleet/issues/23115).

Reviewer: ```suggestion
> Currently, custom packages and Fleet-maintained apps can be automatically installed. macOS App Store apps are [coming soon](https://github.com/fleetdm/fleet/issues/23115).
```

Using "Coming soon" language to be consistent w/ language on [pricing page](https://fleetdm.com/pricing): 
![Screenshot 2024-10-31 at 4 04 54 PM](https://github.com/user-attachments/assets/d9ec5170-a5b1-4cbc-ae31-3fecc8cb93fa)
